sources/settings-extensions/ntp/src/lib.rs

```diff
@@ -85,4 +87,22 @@ mod test {
         let results = serde_json::to_string(&ntp).unwrap();
         assert_eq!(results, test_json);
     }
+
+    fn test_options_ntp() {
+        let test_json = r#"{"time-servers":["https://example.net","http://www.example.com"],"options": ["minpoll", "1", "maxpoll", "2"]}"#;
```

Comment:
For testing `serde_json::to_string()` at the end of this test, you'll need to remove all of the extra whitespace from this string:

```suggestion
        let test_json = r#"{"time-servers":["https://example.net","http://www.example.com"],"options":["minpoll","1","maxpoll","2"]}"#;
```